setup/terminal.md

```diff
@@ -0,0 +1,33 @@
+# Why terminal 
+
+Majority of people when they first start to use the computers they do it through the GUI (Graphical user interface). They do it by clicking or touching things on the screen. 
```

Comment:
The majority of people use GUI (graphical user interface) when they first start to use a computer.